starter-code/jquery_ajax/main.js

```diff
@@ -12,7 +12,20 @@
 'use strict';
 (function() {
   var weatherUrl = "http://api.openweathermap.org/data/2.5/weather?q=";
-  var apiKey = "";
+  var userinput = "city";
+  var appid = "43daa34bfbf43a4d05f3d2b170269cd1";
+  $.ajax ({
+    url : weatherUrl ,
+    type : 'GET',
+    data :  {
+      q: userInput,
```

Comment:
Check the names of variables
```suggestion
      q: userinput,
```